### DISCUSSION TITLE
Quick question on the notebook implementation

### DISCUSSION THREAD

**Discussion by leon-maechler** [FETS-AI/Challenge#121] [Q&A] [answered]
When training, the notebook often gives validation metrics i.e. DICE, hausdorff and so on. 
Is this validation run on the training data or is there an internal train/validate split per datacenter?

If so, what is the train/validate ratio?

Asking so that I can be specific in the manuscript
Thanks!

**Comment by sarthakpati** [comment, marked as answer]
Hi,

The validation will run on a superset of the data as compared with the training data, but the split will be (approximately) 80/20.

Hope this helps!

Cheers,
Sarthak

**Reply by leon-maechler** [reply]
Thanks for the quick reply!

**Reply by sarthakpati** [reply]
No worries, all the best.
